### PR TITLE
[Deepthi M/Abinaya] |GOK-187 | Updated to fetch list of patients with activedrug orders to display in dispense tab.

### PR DESCRIPTION
**Pull request by deepthi-mantena**
## Requirements

- [ ] This PR has a proper title that briefly describes the work done
- [ ] I have squashed / amended the comments to make it more redable
- [ ] I have included link to all the JIRA ticket('s)
- [ ] I wrote the code as a pair or atleast performed a self-review of my own code
- [ ] I have updated the documentation on [Bahmni Wiki](https://bahmni.atlassian.net/wiki/spaces/BAH/overview) or [README](https://github.com/Bahmni/bahmni-lab-frontend/blob/main/README.md) (if applicable)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by deepthi-mantena** [comment]
Screenshot 2023-06-29 at 6 25 14 PM

**Comment by gsluthra** [comment]
Please show some more info here. Name isn't sufficient. There are MANY patients with same name in a given district. Can you add maybe Patient ID, Village, Gender, Age, Mobile number in search results?

**Comment by deepthi-mantena** [comment]
Yes Gurpreet updated to display patient Id also now.Got the similar feedback from desk check aswell.
Screenshot 2023-06-30 at 2 43 49 PM

File: src/inventory/dispense/dispense-page.jsx [hunk 1]
```diff
@@ -13,49 +13,55 @@ import {
...
-                productName: item.name,
-                actualQuantity: item.minimumQuantity
+                patientName: item.name,
+                patientId: item.identifier,
+
             }
         })
 
     }
+    console.log("item...", items);
```
Comment (sikarvarsunil) [inline src/inventory/dispense/dispense-page.jsx:47]: @abinaya-u do we need the console?

**Review by sikarvarsunil — APPROVED** [review]

**Review by gsluthra — APPROVED** [review]

**Comment by gsluthra** [comment]
There are a few extra console.logs ( ). You can keep them for a few more days, until dev is done. But consider removing them once feature is stabilised.

**Comment by deepthi-mantena** [comment]
Sure we will remove them .

**Pull request merged**